component-runtime-manager/src/main/java/org/talend/sdk/component/runtime/manager/ComponentManager.java

```diff
@@ -965,8 +965,7 @@ protected boolean isContainerClass(final Filter filter, final String name) {
                         try {
                             final String strippedJar =
                                     new File(decode(new URL(file.substring(0, separator)).getFile())).getName();
-                            return strippedJar.startsWith("beam-sdks-java-core-")
-                                    || strippedJar.startsWith("org.apache.beam.beam-sdks-java-core-");
+                            return strippedJar.matches(".*beam.sdks.java.core.*");
```

Comment:
Hi ryan

2 notes on that

1. Please no regex, it us too slow at startup
2. Why is it needed? (Assuming pd uses a configurer)